docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<p>Great plot! You immediately wonder... Is something wrong with Probe F? Perhaps worth asking Ben about it (this is independent from this notebook work).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>This is visual coding data so we should ask Josh.</p>